docs/guides/aws-workspace.md

```diff
@@ -258,7 +267,28 @@ As a workaround give the `aws_iam_role` more time to be created with a `time_sle
 
 ```hcl
 resource "time_sleep" "wait" {
-  depends_on = [aws_iam_role.cross_account_role]
+  depends_on = [
+    aws_iam_role.cross_account_role]
   create_duration = "10s"
+}
+```
+
+#### IAM policy error
+
+If you notice below error:
+
+```
+Error: MALFORMED_REQUEST: Failed credentials validation checks: Spot Cancellation, Create Placement Group, Delete Tags, Describe Availability Zones, Describe instances, Describe Instance Status, Describe Placement Group, Describe Route Tables, Describe Security Groups, Describe Spot Instances, Describe Spot Price History, Describe Subnets, Describe Volumes, Describe Vpcs, Request Spot Instances
```

Comment:
this is a duplication of exactly previous section - `Credentials validation checks errors`, which was added by @hamza-tumturk previous release. perhaps, maybe move `time_sleep` resource to main cross-account block, so it's no longer a workaround. I'd suggest to merge these two sections. 

probably we'd also need to remove `MALFORMED_REQUEST: ` from error messages in common/http.go.

Reply:
Errors are nearly the same not the same, the reason I think is in one role is not ready in another role is not right. I am thinking with a few more feedback from users, we should be able to collect all questions and create a FAQ or Troubleshooting section but for the time being, this should work.

Reply:
@tomarv2 , yup, we have internal doc ticket on E2 troubleshooting. will slack you that one.